packages/cli/src/interface/ValidationLogger.ts

```diff
@@ -48,6 +50,19 @@ export default class ValidationLogger {
     this.logUncheckedVars(storageUncheckedVars);
     this.logUninitializedBaseContracts(uninitializedBaseContracts);
     this.logStorageLayoutDiffs(storageDiff, getStorageLayout(this.contract, buildArtifacts));
+    this.logImportsVanillaContracts(importsVanillaContracts);
+  }
+
+  public logImportsVanillaContracts(vanillaContracts: string[] | null): void {
+    if (!vanillaContracts) return;
+    Loggy.noSpin.warn(
+      __filename,
+      'logImportsVanillaContracts',
+      `validation-imports-vanilla-contracts`,
+      `- Contract ${this.contractName} imports ${vanillaContracts.join(
```

Comment:
A bit of a pedantic comment but I do feel it would be best to be accurate here and talk about files importing other files, rather than contracts importing files.

Reply:
There's a problem with that: the validator works on a per-contract basis, not per file. This also means that, if the user has more than one contract defined on the same file that imports vanilla contracts, they'll get the warning twice. 

We could solve this by restructuring how validations work, but given that this is a temporary feature, I preferred to ship anyway. Curious to see what you think though.

Reply:
Agree.